Purchases/Misc/MacDevice.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  MacDevice.swift
+//
+//  Created by Juanpe Catalán on 30/11/21.
+
+#if os(macOS) || targetEnvironment(macCatalyst)
+
+import Foundation
```

Comment:
Nit: this isn't really necessary if you want to remove it.

Reply:
Yep, I'll remove it 👍🏼